# Ros Topics
# image0_topic: "/left_r200/camera/color/image_raw"
# image1_topic: "/right_r200/camera/color/image_raw"

image0_topic: "/test_bot_2/camera/left/image_raw"
image1_topic: "/test_bot_2/camera/right/image_raw"

visualize: true
useGrav: true
# Intrinsic Camera Paramters
fx: 476.7030836014194
fy: 476.7030836014194
cx: 400.5
cy: 400.5
k1: 0.0
k2: 0.0
p1: 0.0
p2: 0.0
k3: 0.0
width: 800
height: 800

# Extrinsic Camera Paramters 
baseline: .025
estimators: [1,2,3]  #1 , 2 ,3 ONLY 1 ESTIMATOR, 4 IS Up down, 5 is all 3
# initX: 3.75
# initY: 0.0
# initZ: 1.25
# initRoll: 0.0
# initPitch: 0.0
# initYaw: 1.570796
initX: 0.0
initY: 0.0 #0
initZ: 0.2090065781259643

initRoll: 0.0
initPitch: 0 # 0.4 #0.62 #0.0
initYaw: 0 # -0.15
gravMag: 0.0
#gravMag: 1.0
